app/views/layouts/_topbar.html.haml

```diff
@@ -0,0 +1,26 @@
+%nav.top-bar{"data-topbar" => "", :role => "navigation"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace